float64[18] q
float64[18] qd
float64[18] trq
float64[9] imu_data   # rpy+gyro+acc
float64[3] body_pos
float64[3] body_vel